source/locale/en/symbols.dic

```diff
@@ -97,7 +96,12 @@ _	line	most
 ▪	black square	some
 ◾	black square	some
 ◦	white bullet	some
-➔	right arrow	some

```

Comment:
Because [U+2794](http://www.fileformat.info/info/unicode/char/2794/index.htm) is not the standard rightwards arrow. IMO the definition was too generic. But if nothing else, it's confusing compared to [U+2192](http://www.fileformat.info/info/unicode/char/2192/index.htm).